test/stdlib/POSIX.swift

```diff
@@ -246,13 +246,7 @@ POSIXTests.test("fcntl(CInt, CInt, UnsafeMutableRawPointer): locking and unlocki
   // Lock for reading...
   var flck = flock()
   flck.l_type = Int16(F_RDLCK)
-  #if os(Android)
-  // In Android l_len is __kernel_off_t which is not the same size as off_t in
-  // 64 bits.
-  flck.l_len = __kernel_off_t(data.utf8.count)
-  #else
   flck.l_len = off_t(data.utf8.count)
```

Comment:
I think the above comment was written before AArch64 support was added, `off_t` actually works fine. This recently broke on Android AArch64 after [an import was added to this file](https://github.com/apple/swift/pull/39591/files#diff-d653aa83a4c23b7858c2e8a84ff427e6e30dca6138928eb2f06808bbd310a12fR7), with the compiler now complaining that `__kernel_off_t` is an `Int` and the count is an `Int64`, but this change fixes it.